core/include/keyman/keyman_core_api.h

```diff
@@ -1140,6 +1140,51 @@ km_core_keyboard_load(km_core_path_name kb_path,
 
 -------------------------------------------------------------------------------
 
+# km_core_keyboard_load_from_blob()
+
+## Description
+
+Parse and load keyboard from the supplied blob and a pointer to the loaded keyboard
+into the out paramter.
+
+## Specification
+
+```c */
+KMN_API
+km_core_status km_core_keyboard_load_from_blob(void* blob, km_core_keyboard** keyboard);
```

Comment:
Ideally should have a length as well to avoid overrun?

Reply:
+1

Reply:
Good point. I guess I worked too much in lazy languages in the last months... :smile: